src/experimental/shader-modules/index.js

```diff
@@ -1,16 +1,15 @@
-import {registerShaderModules} from '../shader-tools';
+// import * as fp64 from './fp64';
 
-import * as fp64 from './fp64';
-
-import * as fog from './fog';
-import * as picking from './picking';
+// import * as fog from './fog';
+// import * as picking from './picking';
 // import * as lighting from './lighting';
-import * as material from './material';
+// import * as material from './material';
 
-registerShaderModules([
-  fp64,
-  fog,
-  picking,
-  // lighting,
-  material
-]);
+// TODO - This probably defeats tree-shaking
```

Comment:
Can we register the modules one by one in their own file so that only importing them will trigger the register?